pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -362,6 +363,7 @@ func (o *InstallOptions) substituteConfigMap(obj *corev1.ConfigMap) *corev1.Conf
 		newContent := strings.ReplaceAll(oldContent, "REVISION", o.Revision)
 		newContent = strings.ReplaceAll(newContent, "NODE_NAME", o.NodeName)
 		newContent = strings.ReplaceAll(newContent, "NODE_ENVVAR_NAME", strings.ReplaceAll(strings.ReplaceAll(o.NodeName, "-", "_"), ".", "_"))
+		newContent = strings.ReplaceAll(newContent, "NODE_ENVVAR_UID", strings.ReplaceAll(strings.ReplaceAll(o.NodeUID, "-", "_"), ".", "_"))
```

Comment:
nit: UID proably needs replacing dashes (-), but not dots (.). 

Example node UID is: ec2b7f7d-12f5-533a-89c6-2f99086de704

Reply:
Good catch. I've updated to just use the uid verbatim.